src/test/java/io/arlas/server/WFSTool.java

```diff
@@ -0,0 +1,9 @@
+package io.arlas.server;
+
+public class WFSTool {
```

Comment:
This class is not direcly linked to WFS stuff.
It should be renamed like `CollectionTool`. This class should wrap code from `AbstractTestWithCollection.beforeClass` and `AbstractTestWithCollection.afterClass`. This code would be called in main method according to args[] values (triggered from maven with something like `-Dexec.args="load"` or `-Dexec.args="delete"`).
`AbstractTestWithCollection` would call `CollectionTool` methodes in `beforeClass` and `afterClass`.